breadcrumbs-ui.js

```diff
@@ -0,0 +1,51 @@
+define([
+    '$'
+], function($) {
+
+    var init = function() {
+        var $breadcrumbs = $('.c-breadcrumbs');
+
+        $breadcrumbs.each(function() {
+            var $this = $(this);
+            var $links = $this.find('.c-breadcrumbs__item');
+            var showMoreText = "Show more breadcrumbs items";
+            var $showMore = $('<li class="c-breadcrumbs__item"><button class="c-breadcrumbs__link c--show-more js-breadcrumb-show" type="button">' + showMoreText + '</button></li>');
+            var containerWidth = $this.width();
+            var linkWidth = 0;
```

Comment:
A better name for this variable might be `contentWidth`, since it’s the total combined width of all the links. As it stands, the name implies this is the width of one link.